### DISCUSSION TITLE
io.quarkus.platform:quarkus-bom vs io.quarkus:quarkus-bom

### DISCUSSION THREAD

**Discussion by Eng-Fouad** [quarkusio/quarkus#29031] [Q&A] [answered]
What is the difference and which one should we use?

Currently, the latest versions in maven repo are:

`io.quarkus.platform:quarkus-bom:2.14.0.CR1`
`io.quarkus:quarkus-bom:2.14.0.Final`

**Comment by gsmet** [comment, marked as answer]
`io.quarkus:quarkus-bom` is the BOM generated by the core release. Except when developing extensions or in very specific cases, you shouldn't use it.

`io.quarkus.platform:quarkus-bom` is the BOM generated by the platform release. It ensures proper alignment with all the Platform members (but only contains the dependencies from the core). This is the BOM you should use for your applications.
